README.md

```diff
@@ -46,6 +46,72 @@ Extend @ni/eslint-config/typescript in the ESLint configuration. Configure the @
 }
 ```
 
+### Angular
+
+ESLint support for Angular is provided by [@angular-eslint](https://github.com/angular-eslint/angular-eslint#readme). It's recommended to use @angular-eslint/schematics to
+configure ESLint for Angular projects especially when migrating from TSLint.
+
+#### Generating a Workspace
+
+Workspaces [generated with @angular-eslint](https://github.com/angular-eslint/angular-eslint#quick-start-with-angular-and-eslint) will generate projects with @angular-eslint.
+
+```bash
+> npm i -g @angular/cli @angular-devkit/{core,schematics} @angular-eslint/schematics
+> ng new --collection=@angular-eslint/schematics
+```
+
+#### Migrating a Workspace
+
+1. Install @angular-eslint/schematics and @ni/eslint-config.
+
+    ```npm install --save-dev @angular-eslint/schematics @ni/eslint-config.```
+
+2. Follow the @angular-eslint [instructions](https://github.com/angular-eslint/angular-eslint#migrating-an-angular-cli-project-from-codelyzer-and-tslint) to run the schematic, listed here for convenience.
+    1. Install the dependencies.
+
+        ```ng add @angular-eslint/schematics```
```

Comment:
Along with @angular-eslint/schematics, this installs packages that may eventually be peer dependencies. I think we should include this documentation ASAP, so we can discuss these now or leave it for later.